TVC_MVP/CountyPresnterProtocol.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CountyPresnterProtocol.swift
+//  TVC_MVP
+//
+//  Created by Shane McCully on 01/03/2019.
+//  Copyright © 2019 Shane McCully. All rights reserved.
+//
+
+import Foundation
+
+protocol CountyPresnterProtocol: class {
+
+    init(view: CountyViewProtocol)
+
+    func viewDidLoad()
+
+    func reloadData()
```

Comment:
We had the chat about this as well. Do you think that it would make sense to have this method visible to the view?